Rakefile

```diff
@@ -4,19 +4,17 @@ $: << File.expand_path('../lib', __FILE__)
 
 require 'bundler/setup'
 
+VENDOR_PATH = File.expand_path('../vendor', __FILE__)
 
 namespace :setup do
   task :test_projects do
     %w[test_project another_test_project].each do |name|
       Dir.chdir("spec/integration/#{name}") do
-        command = (<<-END).lines.map(&:strip).join(' && ')
-        rm -f Gemfile.lock
-        rvm-shell $RUBY_VERSION -c 'rvm gemset create jara-#{name}'
-        rvm-shell $RUBY_VERSION@jara-#{name} -c 'gem install bundler'
-        rvm-shell $RUBY_VERSION@jara-#{name} -c 'bundle install --retry 3'
-        END
+        bundle_path = File.join(VENDOR_PATH, name)
+        command = "bundle install --retry=3 --gemfile=Gemfile --path=#{bundle_path} --binstubs=.bundle/bin"
         puts command
         Bundler.clean_system(command)
+        rm_f '.bundle/config'
```

Comment:
Are you really sure you need to remove this? When I did the same in Puck, I think the only reason why I did this was that there was some test that ensured that no `.bundle/config` was created by Puck, which was hard to verify when it already existed. But my memory rarely serves me correctly so I might be wrong.